capabilities.json

```diff
@@ -3381,6 +3381,9 @@
     }
   ],
   "wasm_abi_versions": [
-    1
+    {
+      "version": 1,
+      "minor_version": 0
+    }
   ]
```

Comment:
I suppose this is less of a worry _now_ since there has not been a release with any ABI version in capabilities.json yet.